src/Transport/AmqpTransport.php

```diff
@@ -29,6 +31,17 @@ public function keepalive(Envelope $envelope, ?int $seconds = null): void
 
             $this->getMessageCount();
         } catch (\Throwable $throwable) {
+            $previous = $this->getTransportException($throwable);
```

Comment:
$previous is misleading - AFIK $throwable can be TransportException as well
$transportException is better